concurrent_server.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import unicode_literals
+from gevent.server import StreamServer
+from gevent.monkey import patch_all
```

Comment:
You probably don't want to import `gevent.patch_all` at module scope.  You wouldn't want someone to stumble across it when importing your handler below from some other code.